Header Header
int32 id
float64 mid_x
float64 mid_y
geometry_msgs/Point[] points